src/pybind/mgr/nfs/tests/test_nfs.py

```diff
@@ -252,7 +252,7 @@ def mock_list_daemons(cls, *args, **kwargs):
                 mock.patch('nfs.ganesha_conf.check_fs', return_value=True), \
                 mock.patch('nfs.export.ExportMgr._create_user_key',
                            return_value='thekeyforclientabc'), \
-                mock.patch('nfs.export.check_cephfs_path'):
+                mock.patch('nfs.export.cephfs_path_is_dir'):
```

Comment:
Fold this commit into 49af0d9704fc59c235f309b49907eb300322ec88 since its related.